{"uuid":"1ea6bc79-bda4-448b-b8cf-2112b6a85576","historyId":"5feda0a8f10c4bbb6d629b5b06ce9a6","fullName":"com.qa.tests.LoginTest.loginTest","labels":[{"name":"package","value":"com.qa.tests.LoginTest"},{"name":"testClass","value":"com.qa.tests.LoginTest"},{"name":"testMethod","value":"loginTest"},{"name":"parentSuite","value":"Default suite"},{"name":"suite","value":"Default test"},{"name":"subSuite","value":"com.qa.tests.LoginTest"},{"name":"host","value":"DESKTOP-9LLI94A"},{"name":"thread","value":"111368@DESKTOP-9LLI94A.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"loginTest","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"org.apache.xmlbeans.XmlException: error: Unexpected end of file after null","trace":"org.apache.poi.POIXMLException: org.apache.xmlbeans.XmlException: error: Unexpected end of file after null\r\n\tat org.apache.poi.xssf.usermodel.XSSFWorkbook.onDocumentRead(XSSFWorkbook.java:299)\r\n\tat org.apache.poi.POIXMLDocument.load(POIXMLDocument.java:159)\r\n\tat org.apache.poi.xssf.usermodel.XSSFWorkbook.<init>(XSSFWorkbook.java:186)\r\n\tat org.apache.poi.ss.usermodel.WorkbookFactory.create(WorkbookFactory.java:73)\r\n\tat com.qa.keywordengine.KeywordEngine.startExecution(KeywordEngine.java:44)\r\n\tat com.qa.tests.LoginTest.loginTest(LoginTest.java:19)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:568)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)\r\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)\r\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)\r\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:794)\r\n\tat org.testng.TestRunner.run(TestRunner.java:596)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:377)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:276)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1134)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1063)\r\n\tat org.testng.TestNG.run(TestNG.java:1031)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)\r\nCaused by: org.apache.xmlbeans.XmlException: error: Unexpected end of file after null\r\n\tat org.apache.xmlbeans.impl.store.Locale$SaxLoader.load(Locale.java:3471)\r\n\tat org.apache.xmlbeans.impl.store.Locale.parseToXmlObject(Locale.java:1270)\r\n\tat org.apache.xmlbeans.impl.store.Locale.parseToXmlObject(Locale.java:1257)\r\n\tat org.apache.xmlbeans.impl.schema.SchemaTypeLoaderBase.parse(SchemaTypeLoaderBase.java:345)\r\n\tat org.openxmlformats.schemas.spreadsheetml.x2006.main.WorkbookDocument$Factory.parse(Unknown Source)\r\n\tat org.apache.poi.xssf.usermodel.XSSFWorkbook.onDocumentRead(XSSFWorkbook.java:256)\r\n\t... 33 more\r\nCaused by: org.xml.sax.SAXParseException; systemId: file:; lineNumber: 1; columnNumber: 1; Unexpected end of file after null\r\n\tat org.apache.xmlbeans.impl.piccolo.xml.Piccolo.reportFatalError(Piccolo.java:1038)\r\n\tat org.apache.xmlbeans.impl.piccolo.xml.Piccolo.parse(Piccolo.java:723)\r\n\tat org.apache.xmlbeans.impl.store.Locale$SaxLoader.load(Locale.java:3439)\r\n\t... 38 more\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1663918220473,"stop":1663918220724}